### DISCUSSION TITLE
官方文档中示意图和描述不一致

### DISCUSSION THREAD

**Discussion by jiaoningbo** [apolloconfig/apollo#4874] [General]
**描述问题**

发现一处文档描述问题，感觉示意图和文字表述的内容不一致

> 后续为了flowchart更简洁，将JVM8080里的内容进行简化，只显示Config Service，里面的Meta Server和Config Service不再显示

https://www.apolloconfig.com/#/zh/deployment/deployment-architecture?id=_222-%e5%8d%95%e6%9c%ba%ef%bc%8c%e5%8d%95%e7%8e%af%e5%a2%83-%e5%88%86%e5%bc%80%e9%83%a8%e7%bd%b2-2%e5%8f%b0linux%e6%9c%8d%e5%8a%a1%e5%99%a8

位置如下：
官方文档 > 部署架构 > 单机 > 单机，单环境 分开部署 2台Linux服务器

**期望**

> 感觉本意为：只显示Config Service，里面的Meta Server和Eureka不再显示

**截图**
<img width="1092" alt="image" src="https://github.com/apolloconfig/apollo/assets/20292655/b826e5cc-4d5e-479a-950e-c508a05ae3fb">

中文文档：https://github.com/apolloconfig/apollo/blob/master/docs/zh/deployment/deployment-architecture.md
英文文档：https://github.com/apolloconfig/apollo/blob/master/docs/en/deployment/deployment-architecture.md

**Comment by Anilople** [comment]
Thanks for finding mistake! 
Could you please submit a pull request with your proposed changes?
